test/scripts/configure-generators

```diff
@@ -31,8 +31,7 @@ generate-build-config-{distro}-{arch}:
     INTERNAL_NETWORK: "true"
   script:
     - sudo ./test/scripts/setup-osbuild-repo
-    - sudo dnf -y install go python3 gpgme-devel s3cmd
-      osbuild osbuild-luks2 osbuild-lvm2 osbuild-ostree osbuild-selinux osbuild-depsolve-dnf
+    - sudo ./test/scripts/install-dependencies
```

Comment:
Not sure if this is a bit of overkill here because it is going to install all the other dependencies too. Or should I split this out a bit further and have a flag to install all and a flag for just this subset?

Reply:
It's a bit of extra work every time we run CI but it's not terrible.  It's fine.